tests/source/issue-5407/one.rs

```diff
@@ -15,6 +15,23 @@ String::new()
 }
 }
 
+// Original from #4649
+trait Foo {
+    fn bar(&self)
+    where
+    //     Self: Bar
+    // Some comment
+    ;
+}
+
+fn foo<T>()
+where
+//     T: Bar,
+// Some comment
+{
+    println!("foo");
+}
+
```

Comment:
Do you think it makes sense to create a `issue_4649.rs`  file to better associate the test case with the original issue?

Reply:
OK. Changed.
Created `issue-4649` **folder** and moved the relevant test cases.  The folder was created to allow testing with both One and Two versions, although I am not sure whether or when it is important to test both versions.